backend/planty/dashboard/views.py

```diff
@@ -245,6 +246,26 @@ def calculate_events(plant: Plant, action: str, last: date, interval: int):
                                  plant.last_fertilized,
                                  plant.instruction.fertilizing)
 
+            # custom events
+            custom_events = CustomEvent.objects.filter(
+                user=user,
+                date__gte=upcoming_start_date,
+                date__lte=upcoming_end_date
+            )
+
+            # FIXME update priority according to the PR with events history
+            for event in custom_events.iterator():
+                events.append({
+                    'date': event.date,
+                    'plant': str(event.plant.id),
+                    'action': 'custom',
+                    'priority': None,
+                    'custom_info': {
+                        'name': event.name,
+                        'description': event.description
+                    }
+                })
```

Comment:
I consider .values() method more readable (at least before we do it properly with e.g. serializers as you mentioned in #20 , but this is good too. It is something to talk about on refactor-meeting if we decide to make one.